lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1158,8 +1158,14 @@ def cmd_route(*args)
             ],
           'ColProps' =>
             {
-              'Subnet'  => { 'MaxWidth' => 17 },
-              'Netmask' => { 'MaxWidth' => 17 },
+              'Subnet'  => {
+                'WordWrap' => false,
+                'MinWidth' => 17
+              },
+              'Netmask' => {
+                'WordWrap' => false,
+                'MinWidth' => 17
+              },
```

Comment:
Assuming this should be updated as well? @adfoster-r7 
```suggestion
              'Subnet'  => { 'Width' => 17 },
              'Netmask' => { 'Width' => 17 },
```